test/__snapshots__/k9.test.ts.snap

```diff
@@ -1,5 +1,213 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`DynamoDBResourcePolicy Typical usage 1`] = `
+Object {
```

Comment:
Jest 28 or 29 finally improved snapshot syntax and removed the type literals everywhere. Much easier on the eyes.